src/main/kotlin/parts/wisdom/utils/RandomLineGenerator.kt

```diff
@@ -0,0 +1,52 @@
+package parts.wisdom.utils
+
+import me.joypri.to
+import parts.wisdom.arcraider.Color
+import parts.wisdom.arcraider.Direction
+import parts.wisdom.arcraider.Length
+import parts.wisdom.arcraider.TheColor
+import parts.wisdom.arcraider.TheDirection
+import parts.wisdom.arcraider.X
+import parts.wisdom.arcraider.Y
+import parts.wisdom.hdezninirola.lineworld.Line
```

Comment:
Breaks layering between our shared `arcraider` infrastructure and a personal sandbox.